core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -407,6 +407,13 @@ public void onLoadChildren(@NonNull String parentId,
     private List<MediaBrowserCompat.MediaItem> loadChildrenSynchronous(@NonNull String parentId) {
         List<MediaBrowserCompat.MediaItem> mediaItems = new ArrayList<>();
         if (parentId.equals(getResources().getString(R.string.app_name))) {
+            long currentlyPlayingMedia = PlaybackPreferences.getCurrentlyPlayingMediaType();
+            if (currentlyPlayingMedia == PlaybackPreferences.PLAYER_STATUS_PLAYING
```

Comment:
Something looks odd here. This retrieves the media *type* (video/audio) but then compares it to a playback *status* (paused/playing).

Reply:
oops, I fixed it with the right function call.